src/features/Common/data/slice.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-param-reassign */
+import { createSlice } from '@reduxjs/toolkit';
+import { RequestStatus } from 'features/constants';
+
+const initialState = {
+  allClasses: {
```

Comment:
why `allClassses` why just  don't `classes`?

Reply:
allClasses stored all the classes list, without pagination. Classes will be used when we include the class page and will be a table with pagination